rdflib/term.py

```diff
@@ -663,6 +672,20 @@ def normalize(self) -> "Literal":
         else:
             return self
 
+    @property
+    def ill_formed(self) -> Optional[bool]:
+        """
+        For `recognized datatype IRIs
+        <https://www.w3.org/TR/rdf11-concepts/#dfn-recognized-datatype-iris>`_,
+        this value will be `True` if the literal is ill formed, otherwise it
+        will be `False`. `Literal.value` (i.e. the `literal value <https://www.w3.org/TR/rdf11-concepts/#dfn-literal-value>`_) should always be defined if this value is `True`, but should not be considered reliable if this property is not `True`.
```

Comment:
Can we double-check the wording at the very end of this line?  I would consider the `.value` not-reliable if this property IS `True`.

Reply:
I think you are right, I will double check and fix it.

Reply:
Yes the documentation's explanation was inverted, should be correct now.